cmake/FindSEACASExodus.cmake

```diff
@@ -0,0 +1,29 @@
+#------------------------------------------------------------------------------#
```

Comment:
Unfortunately, the exodus debian package does not provide exodiff, so on debian/ubuntu one still has to build trilinos ;(